core/src/main/java/org/apache/iceberg/rest/RESTCatalog.java

```diff
@@ -139,61 +138,57 @@ public void createNamespace(Namespace namespace, Map<String, String> metadata) {
         .build();
 
     // for now, ignore the response because there is no way to return it
-    client.post("v1/namespaces", request, CreateNamespaceResponse.class, ErrorHandlers.namespaceErrorHandler());
+    client.post(paths.namespaces(), request, CreateNamespaceResponse.class, ErrorHandlers.namespaceErrorHandler());
   }
 
   @Override
   public List<Namespace> listNamespaces(Namespace namespace) throws NoSuchNamespaceException {
     Preconditions.checkArgument(namespace.isEmpty(), "Cannot list namespaces under parent: %s", namespace);
     // String joined = NULL.join(namespace.levels());
     ListNamespacesResponse response = client
-        .get("v1/namespaces", ListNamespacesResponse.class, ErrorHandlers.namespaceErrorHandler());
+        .get(paths.namespaces(), ListNamespacesResponse.class, ErrorHandlers.namespaceErrorHandler());
```

Comment:
This route is incorrect. List namespaces underneath a parent namespace is `GET /v1/namespaces?parent=accounting`, but this will generate `GET /v1/namespaces/accounting`.

iirc this is because listing the empty namespace is `GET /v1/namspaces`, to make it easier for systems that might _need_ the variable there in the path when it's expected.

Reply:
I think this is correct. We need to support passing parent still, but the path should be the namespaces resource, not the one with a specific namespace.